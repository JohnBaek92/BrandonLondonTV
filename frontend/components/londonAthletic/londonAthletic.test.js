import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import LondonAthletic from "./LondonAthletic";
import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { MemoryRouter } from "react-router";

var jsdom = require("jsdom");

configure({ adapter: new Adapter() });

describe("LondonAthletic component should render as expected", () => {
  const { JSDOM } = jsdom;

  let { document } = new JSDOM("").window;
  let images = {
    dubois: "foo",
    fitzpatrick: "foo",
    lewis: "foo",
    vassel: "foo"
  };
  global.window = {
    ...document
  };
  global.window.images = images;
  it("Renders the LondonAthletic correctly", () => {
    const component = shallow(<LondonAthletic />);
    const tree = toJson(component);

    expect(tree).toMatchSnapshot();
  });
});
