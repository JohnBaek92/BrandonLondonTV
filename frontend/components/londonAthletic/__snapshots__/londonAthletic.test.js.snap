// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LondonAthletic component should render as expected Renders the LondonAthletic correctly 1`] = `
<div
  className="london-athletics-wrapper"
>
  <div
    className="box columns"
  >
    <div
      className="column is-two-fifths"
    >
      <iframe
        className="london-athletic-video"
        frameBorder="1"
        height="100%"
        src="https://www.youtube.com/embed/qUNDomDbYtI"
        width="100%"
      />
    </div>
    <div
      className="column"
    >
      <div
        className="london-athletic-text padding-bottom-london-athletic"
      >
        Through the experience of training for a professional football, Brandon London has positioned himself as one of the top Personal Trainer/Sports Trainers in the nation. With workouts ranging from off-season sports specific training to fitness bootcamps, London Athletic appeals to all.
      </div>
      <div
        className="flex-direction-col"
      >
        <div
          className="padding-bottom-london-athletic font-10px"
        >
          • Strength & Conditioning for Athletes
        </div>
        <div
          className="padding-bottom-london-athletic font-10px"
        >
          • TRX Body Sculpt Classes
        </div>
        <div
          className="padding-bottom-london-athletic font-10px"
        >
          • Fitness Bootcamps
        </div>
        <div
          className="padding-bottom-london-athletic font-10px"
        >
          • Celebrity Training
        </div>
        <div
          className="padding-bottom-london-athletic font-10px"
        >
          • NFL Combine Training
        </div>
        <div
          className="padding-bottom-london-athletic font-10px"
        >
          • Offensive Pass Game Training
        </div>
        <div
          className="padding-bottom-london-athletic font-10px"
        >
          • Power Plate Training
        </div>
        <div
          className="padding-bottom-london-athletic font-10px"
        >
          • Train Like An Athlete Fitness Workout
        </div>
      </div>
    </div>
  </div>
  <div
    className="block clear-both box"
  >
    <div
      className="title is-primary uppercase-center columns is-mobile"
    >
      Catches Clinics - Pass Game Development - Lineman Work
    </div>
    <div
      className="column"
    >
      <div
        className="block flex-row-center"
      >
        <iframe
          frameBorder="1"
          height="auto"
          src="https://www.youtube.com/embed/wk5GSPw_zX8"
          width="100%"
        />
        <iframe
          className="padding-left"
          frameBorder="1"
          height="auto"
          src="https://www.youtube.com/embed/G7-pGlG1DB0"
          width="100%"
        />
      </div>
    </div>
    <div
      className="block flex-row-center"
    >
      <iframe
        frameBorder="1"
        height="auto"
        src="https://www.youtube.com/embed/aAT5wsRrjFs"
        width="100%"
      />
      <iframe
        className="padding-left"
        frameBorder="1"
        height="auto"
        src="https://www.youtube.com/embed/r3xmEm75MUE"
        width="100%"
      />
    </div>
  </div>
  <div
    className="box"
  >
    <div
      className="title is-primary uppercase-center margin-top-point-5"
    >
      High School-NCAA-Pro Athletes Trained
    </div>
    <div
      className="block flex-row-center"
    >
      <img
        className="highschool-photos"
      />
      <img
        className="padding-left highschool-photos"
        src="foo"
      />
      <img
        className="highschool-photos padding-left"
        src="foo"
      />
    </div>
    <div
      className="block flex-row-center"
    >
      <img
        className="highschool-photos"
      />
      <img
        className="padding-left highschool-photos"
        src="foo"
      />
      <img
        className="padding-left highschool-photos"
        src="foo"
      />
    </div>
  </div>
</div>
`;
